### PR TITLE
UHF-251 List of links

### PR DESCRIPTION
**Pull request by teroelonen**
Check the instructions from this pr:
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/28

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/36
https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/14

### CODE REVIEW

File: conf/cmi/core.entity_view_mode.media.list_of_links.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+uuid: 87505fac-4594-40b9-9e84-9b36dc708dae
+langcode: fi
```
Comment (khalima) [inline conf/cmi/core.entity_view_mode.media.list_of_links.yml:2]: Fix me to `langcode: en`.

File: conf/cmi/responsive_image.styles.list_of_links__thumbnail.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+uuid: dc9aa391-887d-4129-b5ed-b30026b1bf77
+langcode: fi
```
Comment (khalima) [inline conf/cmi/responsive_image.styles.list_of_links__thumbnail.yml:2]: This needs to be fixed as well.

**Review by khalima — CHANGES_REQUESTED** [review]
Those pesky langcodes again

File: conf/cmi/core.entity_view_mode.media.list_of_links.yml (hunk 1 above)
Comment (teroelonen) [reply conf/cmi/core.entity_view_mode.media.list_of_links.yml]: Fixed.

File: conf/cmi/responsive_image.styles.list_of_links__thumbnail.yml (hunk 2 above)
Comment (teroelonen) [reply conf/cmi/responsive_image.styles.list_of_links__thumbnail.yml]: Fixed.

**Review by khalima — APPROVED** [review]

**Pull request merged**